### DISCUSSION TITLE
Adding Ghost mode to Kotagram

### DISCUSSION THREAD

**Discussion by charindithjaindu** [kotatogram/kotatogram-desktop#203] [EN 1. General]
if you can, please add the ghost mode like in the mobogram, it would be really useful to users XD
![image](https://user-images.githubusercontent.com/75799963/127696077-0fb1e78b-bb35-456c-9d49-96eca4ae7aa5.png)

**Comment by ilya-fedin** [comment]
It's against https://core.telegram.org/api/terms
